core/src/main/scala/org/scalafmt/config/Newlines.scala

```diff
@@ -36,6 +36,24 @@ import metaconfig.ConfigReader
   *     n =>
   *       consume(n)
   *   }
+  * @param afterCurlyLambda
+  *   If `never` (default), it will remove any extra lines below curly lambdas
+  *   {{{
+  *   something.map { x =>
+  *
+  *     f(x)
+  *   }
+  *   }}}
+  *   will become
+  *   {{{
+  *   something.map { x =>
+  *     f(x)
+  *   }
+  *   }}}
+  *
+  *   If `always`, it will always add one empty line (opposite of `never`).
```

Comment:
That wouldn't be any different than just `force`. 

Part of me wants to leave it as `always` and rely on the fact that, as a general rule in Scalafmt, single-line is always tried before anything else. With that fact in mind `always` (and `force`) make sense.